src/components/TldrawCanvas.tsx

```diff
@@ -1,12 +1,60 @@
-import React, { useRef, useState, useMemo } from "react";
+import React, {
+  useRef,
+  useState,
+  useMemo,
+  useCallback,
+  useEffect,
+} from "react";
 import renderWithUnmount from "roamjs-components/util/renderWithUnmount";
 import isFlagEnabled from "../utils/isFlagEnabled";
-import { Tldraw, TDDocument, TldrawApp } from "@tldraw/tldraw";
+import {
+  App as TldrawApp,
+  defineShape,
+  TLInstance,
+  TLUser,
+  TldrawEditorConfig,
+  Canvas,
+  TldrawEditor,
+  ContextMenu,
+  TldrawUi,
+  TLBaseShape,
+  TLOpacityType,
+  TLBoxUtil,
+  HTMLContainer,
+  TLBoxTool,
+  toolbarItem,
+  MenuGroup,
+  menuItem,
+  OnDoubleClickHandler,
+  TLTranslationKey,
+  OnTranslateEndHandler,
+} from "@tldraw/tldraw";
+import {
+  Button,
+  Classes,
+  Dialog,
+  // Icon,
```

Comment:
I have a local copy of `CytoscapePlayground` that I'm gradually deleting and migrating over time. When that file is empty, migration will be complete. For things I want to leave for later, they've been marked `// TODO`